client/composables/useI18n.ts

```diff
@@ -1,7 +1,7 @@
 import type { DateTimeOptions } from 'vue-i18n'
 import { useI18n as _useI18n } from 'vue-i18n'
-import fr from 'public/assets/locale/fr.json'
-import en from 'public/assets/locale/en.json'
+import fr from '/src/assets/locale/fr.json'
+import en from '/src/assets/locale/en.json'
```

Comment:
seems suspect to me, there should be a special [asset directory](https://nuxt.com/docs/getting-started/assets) at the root of the project. should also be tested in prod mode